README.md

```diff
@@ -1,7 +1,7 @@
 Open Simulation Interface (OSI)
 ===============================
 
-[![Travis Build Status](https://travis-ci.com/OpenSimulationInterface/open-simulation-interface.svg?branch=master)](https://travis-ci.com/OpenSimulationInterface/open-simulation-interface)
+[![ProtoBuf CI Builds](https://github.com/OpenSimulationInterface/open-simulation-interface/actions/workflows/protobuf.yml/badge.svg)](https://github.com/OpenSimulationInterface/open-simulation-interface/actions/workflows/protobuf.yml)
```

Comment:
I don't think those links will work, they should probably be:

```suggestion
[![GitHub Build Status](https://github.com/OpenSimulationInterface/open-simulation-interface/workflows/ProtoBuf%20CI%20Builds/badge.svg)](https://github.com/OpenSimulationInterface/open-simulation-interface/actions?query=workflow%3A%22ProtoBuf+CI+Builds%22)
```

Reply:
This is right out of the Github Generator for Pipeline Badges.
![image](https://user-images.githubusercontent.com/26794482/116378982-e5032b80-a812-11eb-81f5-744cc7518d3c.png)